_docs/setup/kubernetes/helm.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Alpha Quality Helm Chart
```

Comment:
we don't normally put Alpha in doc... the status should be tracked in feature stage doc to be consistent

Reply:
fixed.